pkg/serveswagger/swagger_test.go

```diff
@@ -0,0 +1,28 @@
+package serveswagger
+
+import (
+	"os/exec"
+	"regexp"
+	"testing"
+)
+
+func TestCheckTimestamps(t *testing.T) {
+	exec.Command("go", "generate", "./cmd/go-swagger/restapi")
+	cmd := exec.Command("git", "diff", "../../pkg/serveswagger")
+
+	stdout, err := cmd.Output()
+
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	updates, _ := regexp.Compile(`@@.*@@`)
```

Comment:
Can you please add a comment with a line example that you are trying to match with @@.*@@ regex

Reply:
I'm not sure this will work quite the way you want - there will be additional context lines that don't match this regex and don't begin with `+` or `-`.
What I'd do is something like

```go
lines := strings.Split(string(stdout), "\n")
var changes []string
for _ ,line := range lines {
    if len(line) == 0 {
        continue
    }
    if line[0] == '-' || line[0] == '+' {
        changes = append(changes, line)
    } 
}

// changes now contains every changed/added/removed line, and no
// context lines. Every one of those lines should match the regexp.

generatedLines := regexp.MustCompile(`// Generated at [0-9]{4}-[0-9]{2}`)
for _, line := range changes {
    if !generatedLines.Match(line) {
        t.Errorf("changed line %s does not match regexp",line)
    }
}
```